bokehjs/src/coffee/models/renderers/glyph_renderer.coffee

```diff
@@ -52,6 +52,10 @@ export class GlyphRendererView extends RendererView
     @xscale = @plot_view.frame.xscales[@model.x_range_name]
     @yscale = @plot_view.frame.yscales[@model.y_range_name]
 
+    # compat, deprecated, to be removed
+    @xmapper = @xscale
+    @ymapper = @yscale
+
```

Comment:
My assumption is that the library (bokehjs in this case) uses the new naming convention across the board. Then a chatty approach makes sense to force the user to deal with issues sooner than later. After all, this layer is not provided there to last, but to help user run his code in the mean time of fixing it.